corehq/apps/users/views/__init__.py

```diff
@@ -811,7 +811,7 @@ def make_phone_number_default(request, domain, couch_user_id):
 
     phone_number = request.POST['phone_number']
     if not phone_number:
-        return Http404('Must include phone number in request.')
+        raise Http404('Must include phone number in request.')
```

Comment:
(was tempted to ask for translations, but I think you only see these messages if you're in debug mode)